pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> HashiCorp Vault, a tool for managing secrets.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal a vault, by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
```

Comment:
Don't think that the comment is needed here.

Reply:
Ok, I'll make this shorter.

Reply:
Oops  I meant comma, @Florent2! The comment is needed lol

Reply:
Haha :-) Well, the *comment* is back but the *comma* is gone.

Reply:
Haha :-) Well, the *comment* is back but the *comma* is gone.